### DISCUSSION TITLE
Transparency

### DISCUSSION THREAD

**Discussion by chris-deep** [frang75/nappgui_src#98] [General]
Hi, have you looked at implementing transparent backgrounds for the parent/host window for an app?  Similar to the blend mode / material setting for a ViewController on MacOS.  I know Win10 and later (I guess) support a blended / alpha setting for windows as well.  Not sure about Linux, I guess that depends on GTK and specific Linux version you're running?

**Comment by frang75** [comment]
Hi @chris-deep! This feature is still not implemented. Can be interesting to have some kind of 
`window_alpha(Window *window, const real32_t alpha);`

**Comment by chris-deep** [comment]
Agree...  as I work on NAWV (nAppWebView) I'll dig around and see how I might create a test on Windows for this.   I already know the basics of how to implement on Mac.  I'll drop thoughts here as I have them... Thanks!!
